src/overlays/actors/ovl_En_Xc/z_en_xc.c

```diff
@@ -1746,14 +1746,14 @@ void EnXc_SetThrownAroundSFX(EnXc* this) {
     SkelAnime* skelAnime = &this->skelAnime;
 
     if (Animation_OnFrame(skelAnime, 9.0f)) {
-        func_80078914(&this->actor.projectedPos, NA_SE_PL_BOUND_GRASS);
+        func_80078914(&this->actor.projectedPos, NA_SE_PL_BOUND + SURFACE_SFX_OFFSET_GRASS);
         func_80078914(&this->actor.projectedPos, NA_SE_VO_SK_CRASH);
     } else if (Animation_OnFrame(skelAnime, 26.0f)) {
-        func_80078914(&this->actor.projectedPos, NA_SE_PL_BOUND_GRASS);
+        func_80078914(&this->actor.projectedPos, NA_SE_PL_BOUND + SURFACE_SFX_OFFSET_GRASS);
```

Comment:
Is this kind of change necessary? Why not just have `NA_SE_PL_BOUND_GRASS` ? (idk)

Reply:
Yeah, this one is debatable and I'm still on the fence for it, but I feel it's good to always treat these player sfx id's as `base + materialOffset + ageOffset` because that's how it's designed. A huge number of sfxId's are all labelled `DUMMY_` in the player bank because while they're all valid sfx, their mainly accessed through their base and offset. Just look at the player sfxId's at `0x80 - 0x8F`, `0x90 - 0x9F`, `0xA0 - 0xAF`. But maybe long term when we properly name all sfxId's, we will be giving them all names based on combinations of `base + material offset + age offset`

Reply:
The key thing that bothers me is as you said

> these player sfx id's

so I understand it in the context of player, but this is just an actor randomly using a player sfx for its own purpose, it doesn't care all that much about the layout of the player bank?
(I'm on the fence too)

Reply:
Actually this specific actor is even weirder because it does use `SurfaceType_GetSfxOffset` so it's a poor example
The point stands for others though

Reply:
Indeed. These "player sfxIds" are called "player" because the og name for this was "Player Bank". However, as you mentioned, some actors use these for themselves (non-player actors), like walking most commonly, but other sfx as well

Reply:
This actor specifically (Sheik) uses a hardcoded sfxId/material/offset because it's cutscene based, so it's all predetermined where she'll be walking